src/limbo/bayes_opt/boptimizer.hpp

```diff
@@ -35,18 +35,18 @@ namespace limbo {
         /**
         The classic Bayesian optimization algorithm.
 
-        \rst
+        \\rst
```

Comment:
Ooops. I had an issue with my linux installation and had to retrieve both files with a recovery tool. It changed enters with literals \n and quotes with \" so I replaced them, but I didn't noticed this. I will fix them ASAP